src/components/Selector/Selector.test.tsx

```diff
@@ -0,0 +1,124 @@
+import "@testing-library/jest-dom/extend-expect"
+import React, { useState } from "react"
+import { fireEvent, render } from "../../tests/utils"
+
+import { Selector } from ".."
+import { mockedOptions } from "./__mocks__/options"
+import { SelectedItems, SelectorProps } from "./Selector"
+
+const ControlledMultiSelectorTemplate = (props: SelectorProps) => {
+  const [selectedItems, setSelectedItems] = useState<SelectedItems>([])
+
+  return (
+    <Selector
+      selectedItems={selectedItems}
+      multi={true}
+      {...props}
+      onSelectedItemsChange={(selectedItems) => {
+        setSelectedItems(selectedItems)
+        props.onSelectedItemsChange?.(selectedItems)
+      }}
+    />
+  )
+}
+
+describe("Selector", () => {
+  it("renders successfully", () => {
+    const { getByTestId } = render(<Selector options={mockedOptions} />)
```

Comment:
Avoid using _ByTestId queries if possible, treat it as a last resort if there's no other way to get it with the other queries, as it doesn't represent how the user will interact with it very well.

In this specific case you could use getByRole("button", { name: "Select an option" }) for example, or getByLabelText if the label exists and it points to the correct element

Same goes for the other uses of _ByTestId in the tests.

Reference: https://testing-library.com/docs/queries/about/#priority